### PR TITLE
Digital User Engagement Events dashboard

### PR DESCRIPTION
**Pull request by satyaso**
*New AWS Sample - DUE*

*Digital User Engagement Events dashboard Template*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -657,3 +657,30 @@ Chaining the [Amazon S3 Triggered Endpoint Imports](#Amazon-S3-Triggered-Endpoin
 ## License
 
 This library is licensed under the MIT-0 License. See the LICENSE file.
+
```
Comment (davelemons) [inline README.md:687]: Can you please move this under the: Amazon Pinpoint Basics section making sure to also include it in the Table of Contents above?  I tend to share the Table of Contents with customers so they have an idea of the types of things that are in this repo

**Review by davelemons — CHANGES_REQUESTED** [review]
See my comment about moving Readme text into Pinpoint Basics section and updating TOC.

**Comment by satyaso** [comment]
I have updated the readme file as suggested.

**Review by davelemons — APPROVED** [review]
Looks Great

**Pull request merged**